packages/analytics-types/src/storage.ts

```diff
@@ -1,10 +1,13 @@
+import { Logger } from './logger';
+
 export interface Storage<T> {
   isEnabled(): Promise<boolean>;
   get(key: string): Promise<T | undefined>;
   getRaw(key: string): Promise<string | undefined>;
   set(key: string, value: T): Promise<void>;
   remove(key: string): Promise<void>;
   reset(): Promise<void>;
+  logger?: Logger;
```

Comment:
are we able to avoid this? would prefer that we do not touch public interfaces

Reply:
Removed it